### PR TITLE
feat(http): add server headers to response

### LINKED ISSUE #98: [sdk-middleware-http] Include headers in request response
### Description

The headers object is missing from the response.

### Expected Behavior

Should include the headers in the response
### Current Behavior

the response header is striped out.

### PR DESCRIPTION
**Pull request by hisabimbola**
#### Summary

Fixes #98 
#### Todo

- Tests
    - [x] Unit
    - [ ] Integration
    - [ ] Acceptance
- [x] Documentation

### CODE REVIEW

**Comment by hisabimbola** [comment]
I was going through the previous implementation [here](https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/services/base.coffee#L583) and I think we should probably implement this to look something like that

**Comment by hisabimbola** [comment]
Hello, the build is failing because though _raw()_ method is present in the actual implemented, somehow it's not present in the flow definition hence the error.

Digging further into the error made me to discover that the node implementation of the fetch API is behind the official implementation. Some method like _raw_ that is the node version is not in the browser 😓

File: packages/sdk-middleware-http/src/http.js [hunk 1]
```diff
@@ -50,6 +50,8 @@ export default function createHttpMiddleware (
               body: result,
               statusCode: res.status,
             }
+            if (options.includeHeaders)
+              parsedResponse.headers = res.headers.raw()
```
Comment (emmenko) [inline packages/sdk-middleware-http/src/http.js]: Use `parseHeaders`

File: packages/sdk-middleware-http/test/http.spec.js [hunk 2]
```diff
@@ -48,6 +48,65 @@ describe('Http', () => {
...
+          'Content-Type': 'application/json',
+        })
+        .get('/foo/bar')
+        .reply(200, { foo: 'bar' })
+
+      httpMiddleware(next)(request, response)
+    }),
+  )
+
+  it('should omit the headers when headers flag is false or omitted', () =>
```
Comment (emmenko) [inline packages/sdk-middleware-http/test/http.spec.js]: You don't need this test

Comment (wizzy25) [inline packages/sdk-middleware-http/test/http.spec.js]: Maybe to ensure in doesn't throw/fail if omitted??

Comment (emmenko) [inline packages/sdk-middleware-http/test/http.spec.js]: It's the same test as `execute a get request (success)` (L24)

File: packages/sdk-client/src/client.js [hunk 3]
```diff
@@ -43,6 +43,7 @@ export default function createClient (options: ClientOptions): Client {
             rs.resolve({
               body: rs.body || {},
               statusCode: rs.statusCode,
+              headers: rs.headers || {},
```
Comment (emmenko) [inline packages/sdk-client/src/client.js]: Test is missing

**Comment by hisabimbola** [comment]
> Also make sure to update the documentation (e.g. sdk-client)

I read through the documentation and did not see anything to update, maybe something else was desired. I added documentation about the new flag in the http middleware docs

**Comment by hisabimbola** [comment]
Also, I am planning to add `maskSensitiveHeaderData` flag.

**Comment by hisabimbola** [comment]
@emmenko Improved the feature. Please review again.

**Pull request merged**